package.json

```diff
@@ -10,7 +10,8 @@
     "lint:fix": "eslint --fix . --ext .ts,.tsx,.js",
     "stats": "NODE_ENV=production webpack --mode production --env.stats --profile --json > stats.json && webpack-bundle-analyzer ./stats.json ./dist",
     "test": "echo Testing... && echo OK",
-    "verify": "npm run lint && npm run test"
+    "verify": "npm run lint && npm run test",
+    "format": "prettier --write \"src/**/*.ts\" \"src/**/*.js\""
```

Comment:
Should this run as part of the hooks?

Reply:
The lint already checks that the code is OK, we autoformat in the code. 
This format won't be used unless copy pasting code/files for example